Packs/PrismaAccess/Playbooks/Prisma_Access_Whitelist_Egress_IPs_on_SaaS_Services/README.md

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
This playbook missing description/overview

Reply:
Fixed